common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -81,3 +92,89 @@ def test_dashboard_social_sharing_feature(self):
         self.assertEqual(facebook_widget.attrs('target')[0], '_blank')
         self.assertIn(facebook_url, facebook_widget.attrs('href')[0])
         self.assertIn(facebook_url, facebook_widget.attrs('onclick')[0])
+
+    def test_ended_course_date(self):
+        """
+        Validate the behavior of course dates on student dashboard, when course has ended
```

Comment:
@saleem-latif doc string of test case should be very verbose. It should explain every step involved. You can see the doc strings of tests we written for certificates i.e test_page_has_accomplishments_banner.